src/cloud-event.ts

```diff
@@ -53,5 +58,9 @@ export interface SalesforceFunctionContext {
 }
 
 function parseBase64Json<A>(data: string): A {
-  return JSON.parse(Buffer.from(data, "base64").toString("utf-8"));
+  try {
+    return JSON.parse(Buffer.from(data, "base64").toString("utf-8"));
+  } catch (error) {
+    throw new Error(error.message);
```

Comment:
Could we add some helpful context here? I think this is just going to show a `Unexpected end of JSON input` type message which won't give the developer much of a clue to the problem. Maybe something more like ``Could not execute function. Function arguments could not be determined due to invalid JSON body: ${error.message}``.